tools/iptb-plugins/filecoin/local/output.go

```diff
@@ -0,0 +1,75 @@
+package pluginlocalfilecoin
+
+import (
+	"context"
+	"io"
+	"os/exec"
+)
+
+type Output struct {
+	args     []string
+	err      error
+	exitcode int
+	exitset  bool
```

Comment:
What does this do? Based on this file it looks like it might not be needed.

Reply:
It's used to short cut in the `ExitCode` function after it's been called once so we don't call `cmd.Wait()` twice.